src/main/kotlin/com/yapp/demo/oauth/utils/TokenUtils.kt

```
@@ -0,0 +1,21 @@
+import java.util.Base64
+
+fun parseHeaders(idToken: String): Map<String, String> {
+    if (idToken.isBlank() || idToken.count { it == '.' } < 2) {
+        throw CustomException(ErrorCode.TOKEN_INVALID)
+    }
+
+    return try {
+        val encodedHeader = idToken.substringBefore(".")
+        val decodedHeader = String(Base64.getUrlDecoder().decode(encodedHeader))
+        jacksonObjectMapper().readValue(decodedHeader, object : TypeReference<Map<String, String>>() {})
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)